packages/ui/src/theme/tokens/components/switchField.ts

```diff
@@ -36,14 +17,45 @@ export const switchfield = {
   track: {
     backgroundColor: { value: '{colors.background.tertiary.value}' },
     borderRadius: { value: '{radii.xxxl.value}' },
-    checked: {
-      backgroundColor: { value: '{colors.brand.primary.60.value}' },
-    },
     height: { value: '{space.relative.medium.value}' },
     padding: { value: '{outlineWidths.medium.value}' },
     transition: {
       duration: { value: '{time.short.value}' },
     },
     width: { value: '{space.relative.xl.value}' },
   },
+
+  // States
+  _disabled: {
+    opacity: { value: '{opacities.60.value}' },
+  },
+  _focus: {
+    track: {
+      shadow: {
+        value: {
+          offsetX: '0px',
+          offsetY: '0px',
+          blurRadius: '0px',
+          spreadRadius: '2px',
+          color: '{colors.border.focus.value}',
+        },
+      },
+    },
+  },
+  _checked: {
```

Comment:
FYI, pushed a commit that updates 'checked' to '_checked' to follow our _state guidance, and so I can use this as an example in docs for token definitions :)

Reply:
Reverted this change to avoid breaking change to property names.